src/init.f90

```diff
@@ -398,8 +430,10 @@ subroutine param_assign(run_params, lowerbounds, upperbounds, nDerived, paramsPl
        run_params%subpopNP = run_params%DE%NP !no partitioning, so the subpopulation is the same as the whole population
     endif
 
-    !Default is to output parameter samples.
-    call set_logical(run_params%outputSamples, .true., invar=outputSamples)
+    !Default is to enable all output files.
+    call set_logical(run_params%disableIO, .false., invar=disableIO)
```

Comment:
It does.  Good point about needing to be a bit more transparent with the user about this case.  I think a warning is probably less annoying than making people update their values of outputRaw and outputSam when they set disableIO true, so I've added that now.  I've also now made it so that outputRaw and outputSam default to the opposite of disableIO, so that in the absence of explicit instructions, .sam and .raw file output behaviour just follows what you'd expect based on the value of disableIO.

Reply:
This is much clearer, thank you!